ckanext/unaids/plugin.py

```diff
@@ -297,7 +297,7 @@ def _data_dict_is_resource(data_dict):
 def _giftless_upload(context, resource, current=None):
     attached_file = resource.pop("upload", None)
     if attached_file:
-        if type(attached_file) == FlaskFileStorage:
+        if type(attached_file) is FlaskFileStorage:
```

Comment:
It was oddly reported by linter. I do not remember why I went with `==` instead of `isinstance` (I wrote this piece ages ago) and didn't want to risk of introducing a bug.
`is` is almost identical to `==` and it made linter happy.